gradle/mavenPublish.gradle

```diff
@@ -69,15 +69,6 @@ publishing {
                     url = 'https://github.com/gluonhq/attach'
                 }
             }
-
-            pom.withXml {
-                Node pomNode = asNode()
-                pomNode.dependencies.'*'.findAll() {
-                    it.groupId.text() == 'org.openjfx'
-                }.each {
-                    it.parent().remove(it)
-                }
-            }
```

Comment:
I published the artifacts to my local and they had an entry for javafx dependency. can you verify?

From `accelerometer-4.0.20-SNAPSHOT.pom` : 

```
<dependency>
  <groupId>org.openjfx</groupId>
  <artifactId>javafx-graphics</artifactId>
  <version>20.0.2</version>
  <scope>runtime</scope>
</dependency>
```

Reply:
yes, there is an entry, but there is no javafx classifier (which was the main purpose of removing them from the pom).
I don't see an issue with having a real JavaFX runtime dependency anymore

Reply:
I am aware these are empty jars. However, the projects using Attach will still have to declare a javafx dependency and this might pollute the dependency tree.

Reply:
Any project that uses Attach _needs_ JavaFX dependencies nonetheless, so that shouldn't be a problem. Quite the contrary: if a non-JavaFX project uses Attach (not sure if this is possible!?) without adding JavaFX dependencies, it will fail. Technically, Attach uses JavaFX as runtime dependency, and we started removing it from the pom because of the classifier issues, and then because of version conflicts solving those classifiers. My take is that with the new JavaFX gradle plugin, that won't be an issue anymore.